src/services/schedule-services.js

```diff
@@ -0,0 +1,66 @@
+import {
+	addSoldiersToDuty,
+	getAvailableSoldiersForDuty,
+	getUnscheduledDuties,
+} from "../db/duty-collection.js";
+import { sortSoldiersAccordingToScore } from "../db/justice-board-collection.js";
+
+function canScheduleDuty(duty) {
+	return !(
+		duty.status === "scheduled" ||
+		duty.status === "canceled" ||
+		new Date(duty.startTime) < new Date()
```

Comment:
`startTime` should already be a date

```suggestion
		duty.startTime.getTime() < Date.now()
```